leonzwagner/my-portofolio
# Navbar

### Description

I propose to create responsive navbar, and implemented `Atomic Design` to make the component reusable.

### Issue

close #5 (Feature enhancment with Responsive navbar)

### Steps to Test

1. Ensure that all existing features are functioning correctly
2. Check /src dir to verify that all Navbar components are correctly existed.
3. Follow these steps:
- Import `./src/components/Fragment/NavbarMobile` in App.tsx
- Open up browser to ensure it rendered properly.

### Checklist

- [] Tests pass (if applicable)

### Additional Notes

Do `npm install` or `pnpm install` to update your local dependencies.
